lib/yabeda/sidekiq.rb

```diff
@@ -104,5 +108,23 @@ def custom_tags(worker, job)
         worker.method(:yabeda_tags).arity.zero? ? worker.yabeda_tags : worker.yabeda_tags(*job["args"])
       end
     end
+
+    self.job_tags = []
```

Comment:
What about using `Set` for this? And may be choosing more descriptive name :thinking: